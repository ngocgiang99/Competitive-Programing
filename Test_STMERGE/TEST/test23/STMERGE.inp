4
6 10
271695680 668410180 635356084 665759880 731024798 541590025 401843629 193455881 882941267 999016138 
526014792 810131734 992931845 646491322 197695050 811664903 485885337 451984720 895891640 100926830 
225037363 38176319 711272663 71287586 543189561 172445053 929129261 667385656 9318479 259476287 
941837950 48112131 539403611 826138106 984022991 106945051 587638914 265066595 965936229 219485266 
964220201 403603822 79051205 618697663 555375860 361059410 213560756 488064950 915364808 22677122 
301918776 272843725 939398092 231539636 26360212 165566424 163422051 532282846 263182927 734266349 
5 1
241737517 
775751264 
780438048 
727020585 
748279091 
10 6
962945397 334024102 874703093 427488807 981650955 489667083 
121622273 962864716 917214498 447920167 820551999 491350945 
925240366 447989760 885687109 222394446 257391843 794665434 
302162664 510026422 272087351 863825466 980334139 799951231 
828446560 409320421 684324757 810773901 538304465 724484576 
505102731 18180102 354021184 572349808 584979184 35253327 
686808738 989802010 98586844 165258764 110081835 111812516 
441487521 794335011 934493778 151859730 610897878 886490553 
763472843 847476136 878581328 841362994 163396490 793788653 
420192290 468719412 168673733 750333605 890609572 218237348 
2 10
200731496 251828902 872339908 902667203 203802181 183139232 984671025 856276467 754301263 669209142 
745861742 545548841 958723548 818861646 502266096 402913388 982553472 150934541 521287787 65805568 
