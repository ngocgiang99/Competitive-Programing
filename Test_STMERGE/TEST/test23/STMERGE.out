301918776
241737517
420192290
266537064
